mongo/integration/index_view_test.go

```diff
@@ -29,35 +29,44 @@ func TestIndexView(t *testing.T) {
 
 	mt.Run("list", func(mt *mtest.T) {
 		verifyIndexExists(mt, mt.Coll.Indexes(), index{
-			Key: map[string]int{
-				"_id": 1,
-			},
+			Key:  bson.D{{"_id", int32(1)}},
 			Name: "_id_",
 		})
 	})
 	mt.RunOpts("create one", noClientOpts, func(mt *mtest.T) {
-		mt.Run("success", func(mt *mtest.T) {
+		mt.Run("name not specified", func(mt *mtest.T) {
```

Comment:
We weren't actually asserting that the generated name was correct, so I added an assertion to this test.